modules/nodes-descheduler-filtering-priority.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-descheduler.adoc
+
+[id="nodes-descheduler-filtering-priority_{context}"]
+= Filtering pods by priority
+
+You can configure descheduler strategies to consider pods for eviction only if their priority is lower than a specified priority level. Pods that are higher than the specified priority threshold are not considered for eviction.
+
+You can use the `ThresholdPriority` parameter to set a numerical priority threshold, or you can use the `ThresholdPriorityClassName` parameter to specify a certain priority class name.
+
+.Prerequisites
+
+* Cluster administrator privileges.
+
+.Procedure
+
+. Edit the `KubeDescheduler` object:
++
+[source,terminal]
+----
+$ oc edit kubedeschedulers.operator.openshift.io cluster -n openshift-kube-descheduler-operator
+----
+
+. Add either the `ThresholdPriority` or `ThresholdPriorityClassName` parameter to one or more strategies:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1beta1
+kind: KubeDescheduler
+metadata:
+...
+spec:
+  deschedulingIntervalSeconds: 3600
+  strategies:
+  - name: "RemovePodsHavingTooManyRestarts"
+    params:
+    - name: "PodRestartThreshold"
+      value: "10"
+    - name: "IncludingInitContainers"
+      value: "false"
+    - name: "ThresholdPriority" <1>
+      value: "10000"
+  - name: "PodLifeTime"
+    params:
+    - name: "MaxPodLifeTimeSeconds"
+      value: "86400"
+    - name: "ThresholdPriorityClassName" <1>
+      value: "my-priority-class-name" <2>
+----
+<1> You cannot specify both `ThresholdPriority` and `ThresholdPriorityClassName` for the same strategy.
+<2> The numerical priority value associated with this priority class name is used as the threshold.
+
+. Save the file to apply the changes.
```

Comment:
May be it is good to add the below point here ?
 if the given priority class does not exist, descheduler won't create it and will throw an error.

Reply:
Thanks !!